core/modules.rs

```diff
@@ -893,9 +818,12 @@ impl RecursiveModuleLoad {
     &mut self,
     scope: &mut v8::HandleScope,
     module_request: &ModuleRequest,
-    module_source: &ModuleSource,
+    module_source: ModuleSource,
```

Comment:
Note that I left this struct copy in as it prevents clones later on, and the nesting is pretty shallow.